src/utils.js

```diff
@@ -0,0 +1,23 @@
+export const userName = document
+	.querySelector("head")
+	.getAttribute("data-user-displayname");
```

Comment:
`.getAttribute("data-user-displayname")` should be `.getAttribute("data-user");`